Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp

```diff
@@ -1029,7 +1035,7 @@ IPCCommandResult CWII_IPC_HLE_Device_net_ip_top::IOCtl(u32 _CommandAddress)
 			);
 
 			// Do not pass return-only events to the native poll
-			ufds[i].events &= ~(POLLERR | POLLHUP | POLLNVAL);
+			ufds[i].events &= ~(POLLERR | POLLHUP | POLLNVAL | FIX_WSAPOLL);
```

Comment:
How about a different name? `FIX_` sounds a bit odd (and doesn't indicate the purpose without looking at the definition), maybe `UNSUPPORTED_` or something like that?

And if we decide to do that, it might be a good idea to put a note next to the WIN32 define that briefly explains this (or links to the according docs...or both) for documentation purposes.